sopel/run_script.py

```diff
@@ -148,19 +149,22 @@ def build_parser():
 
 
 def check_not_root():
+
```

Comment:
I think you should remove this blank line. It makes the docstring for this function into a random multiline string/comment.

Reply:
removed the blank line